.github/workflows/release.yml

```diff
@@ -111,6 +114,11 @@ jobs:
               './kwctl-windows-x86_64-sbom/kwctl-windows-x86_64-sbom.spdx',
               './kwctl-windows-x86_64-sbom/kwctl-windows-x86_64-sbom.spdx.cert',
               './kwctl-windows-x86_64-sbom/kwctl-windows-x86_64-sbom.spdx.sig',
+              './kwctl-linux-x86_64.attestation.sigstore.json//kwctl-linux-x86_64.attestation.sigstore.json',
```

Comment:
```suggestion
              './kwctl-linux-x86_64.attestation.sigstore.json/kwctl-linux-x86_64.attestation.sigstore.json',
```

Reply:
Not present, as we aren't uploading those artifacts anymore.